cmd/agent/common/loader.go

```diff
@@ -30,6 +30,9 @@ func LoadComponents(confdPath string) {
 	if flavor.GetFlavor() != flavor.ClusterAgent {
 		workloadmeta.GetGlobalStore().Start(context.Background())
 
+		// uncomment this to dump all workloadmeta events as debug logs
+		//dumper.Enable()
```

Comment:
This was intended as a debugging tool.  It helped me see some events, and I was imagining myself adding a collector some new service and wanting to compile this in so I could eyeball the events my new code was generating.  So, more than telemetry, and more than `workload-list`.

So it's not something that would be enabled on a running agent, and I put it in a different package to avoid adding package size.  I'd be happy to remove this entirely -- I figured since I'd written it I might as well see if we wanted to keep it :)